tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockFileSystemTests.cs

```diff
@@ -103,20 +103,14 @@ public void MockFileSystem_AddFile_ShouldHandleNullFileDataAsEmpty()
         }
 
         [Test]
-        public void MockFileSystem_AddFile_ShouldReplaceExistingFile()
```

Comment:
Did you delete this deliberately?

Reply:
No, this was accidental. I have restored it. Thank you for catching this!